<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetAllTopicDlg</class>
 <widget class="QDialog" name="GetAllTopicDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Get All Topics</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>11</y>
     <width>371</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Select Topic ID</string>
      </property>
      <widget class="QComboBox" name="Select_Topic_ID">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Select Topic ID</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="title">
       <string>Name of Selected Topic</string>
      </property>
      <widget class="QLabel" name="Name_Topic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>351</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="title">
       <string>Description of Selected Topic</string>
      </property>
      <widget class="QLabel" name="Description_Topic">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>351</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Close_Get_All_Topic">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
